diff --git a/node_modules/@types/chai/index.d.ts b/node_modules/@types/chai/index.d.ts
index a813adf..26c792f 100755
--- a/node_modules/@types/chai/index.d.ts
+++ b/node_modules/@types/chai/index.d.ts
@@ -20,6 +20,16 @@ declare namespace Chai {
     type Message = string | (() => string);
     type ObjectProperty = string | symbol | number;
 
+    // Self define
+    type ResponseString = `${number}:${string}`;
+
+    export interface ExpectStatic {
+        (val: any, message?: ResponseString): Assertion;
+        fail(message?: ResponseString): never;
+        fail(actual: any, expected: any, message?: ResponseString, operator?: Operator): never;
+    }
+    // -----------------------
+
     interface PathInfo {
         parent: object;
         name: string;
@@ -102,11 +112,11 @@ declare namespace Chai {
         version: string;
     }
 
-    export interface ExpectStatic {
-        (val: any, message?: string): Assertion;
-        fail(message?: string): never;
-        fail(actual: any, expected: any, message?: string, operator?: Operator): never;
-    }
+    // export interface ExpectStatic {
+    //     (val: any, message?: string): Assertion;
+    //     fail(message?: string): never;
+    //     fail(actual: any, expected: any, message?: string, operator?: Operator): never;
+    // }
 
     export interface AssertStatic extends Assert {
     }
